### ISSUE TITLE
Support having multiple Julia binaries

### ISSUE THREAD

**Issue by under-Peter** [JunoLab/Juno.jl#172]
It would be nice if Juno offered the possibility to have multiple paths for julia and an associated drop-down menu to chose which binary to use.
Use cases involve quickly benchmarking different versions of Julia in Juno, switching to 0.7 for deprecation warnings if 1.0 doesn't work as expected and working with projects that support different versions of julia.

Cheers

**Comment by crstnbr** [comment]
+1 for this. Would also be nice for switching to the current nightly or a julia with `-J mycustomsysimg.dll`.

I reckon this should be rather trivial to implement, no?

**Comment by crstnbr** [comment]
I would really love to see this. Quite regularily I find myself in a position where I want to use a different Julia binary in Juno than the default one. A few scenarios are switching between

* different Julia versions (0.7 for example for porting old code)
* regular Julia and one with a custom image
* binaries from the webpage (OpenBLAS) and a self-compiled version with MKL
* stable "production" Julia and a development build

I think it would be great if we could allow users to specify multiple Julia binaries and have them switch between them using a, say, `Julia Client: Select Julia Binary` option.

**Comment by pfitzseb** [comment]
It would be enough to have a custom start command (binary location + cmdline args) for this, right?

**Comment by crstnbr** [comment]
Yes. And perhaps a custom name field to give the binaries nice names like "Julia 1.3.1 MKL".
